generic/examples/simple.py

```diff
@@ -0,0 +1,87 @@
+# Grid size including IOBs at edges
+X = 12
+Y = 12
+# SLICEs per tile
+Z = 8
+# LUT input count
+K = 4
+# Number of local wires
+L = Z*(K+1) + 8 
```

Comment:
You might want to consider using the term `W` to represent some form of channel width -- i.e. the number of routing tracks in a routing channel; `L` is typically used to describe the length of a routing wire